### PR TITLE
SASS-7605: Upgraded to Play 3.0

### PR DESCRIPTION
**Pull request by ansahdavid**
Upgrading income-tax-submission-frontend to play 3.0

Linked to Jira ticket SASS-7605

##### Before Reviewing
- [ ]  Have you pulled the branch down?
- [ ]  Have you assigned yourself to the PR?
- [ ]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: app/controllers/testOnly/UserResearchLoginController.scala [hunk 1]
```diff
@@ -56,6 +56,7 @@ class UserResearchLoginController @Inject()(mcc: MessagesControllerComponents,
   private[testOnly] def buildGGSession(authExchange: AuthLoginAPIResponse): Session = Session(Map(
     SessionKeys.sessionId -> (if (authExchange.sessionId.isEmpty) SessionId(s"session-${UUID.randomUUID}").value else authExchange.sessionId),
     SessionKeys.authToken -> authExchange.token,
-    SessionKeys.lastRequestTimestamp -> DateTime.now.getMillis.toString
+//    SessionKeys.lastRequestTimestamp -> DateTime.now.getMillis.toString
```
Comment (basotia) [inline app/controllers/testOnly/UserResearchLoginController.scala:59]: can we please remove this commented line

File: README.md [hunk 2]
```diff
@@ -19,60 +17,65 @@ Run the following command to start the remaining services locally:
...
-| Identifier Value 1  | 1234567890                                                                        |
-| Enrolment Key 2     | HMRC-AS-AGENT                                                                     |
-| Identifier Name 2   | AgentReferenceNumber                                                              |
-| Identifier Value 2  | XARN1234567                                                                       
+| FieldName            | Value                                                                             |
+|----------------------|-----------------------------------------------------------------------------------|
+| Redirect url         | /test-only/2022/additional-parameters?ClientNino=AA123457A&ClientMTDID=1234567890 |
+| Credential Strength  | weak                                                                              |
+| Confidence Level      | 250                                                                               |
+| Affinity Group        | Agent                                                                             |
```
Comment (basotia) [inline README.md:72]: can we please align them

File: app/controllers/testOnly/UserResearchLoginController.scala (hunk 1 above)
Comment (ansahdavid) [reply app/controllers/testOnly/UserResearchLoginController.scala:59]: Will remove in the next commit

File: README.md (hunk 2 above)
Comment (ansahdavid) [reply README.md:72]: This is already aligned

**Review by AishaBi — DISMISSED** [review]

**Review by MadStu — DISMISSED** [review]

**Review by AishaBi — DISMISSED** [review]

**Review by MadStu — DISMISSED** [review]

**Review by AishaBi — APPROVED** [review]

**Review by MadStu — APPROVED** [review]

**Pull request merged**